Libplanet/Crypto/PublicKeyGetter.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+
+namespace Libplanet.Crypto
+{
+    /// <summary>
+    /// A static getter class for determine which public key it is by given <see cref="byte"/>
+    /// representation and returns a concrete class type in <see cref="IPublicKey"/>.
+    /// </summary>
+    public static class PublicKeyGetter
+    {
+        /// <summary>
+        /// Gets a concrete class in <see cref="IPublicKey"/> by checking given <see cref="byte"/>
+        /// representation length.
```

Comment:
How about specifying the length and its return type?

Reply:
https://github.com/planetarium/libplanet/blob/1f83dee55e5ed0f64d72da1c5c620b01b17eb513/Libplanet/Crypto/PublicKeyGetter.cs#L12-L25
Amended!